lib/java/com/google/android/material/internal/FlowLayout.java

```diff
@@ -164,16 +209,77 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
       if (i == (getChildCount() - 1)) {
         maxChildRight += rightMargin;
       }
+
+      remainingItems--;
     }
 
     maxChildRight += getPaddingRight();
     childBottom += getPaddingBottom();
 
+    // if we have remaining child views, adjust height to enable showing the `X more child` view on
+    // next line
+    if (this.overflowChildEnabled && remainingItems > 0) {
+      View child = this.getChildAt(this.getChildCount() - 1);
```

Comment:
This alters the text of the last child view so that the user cannot display the original context, this is not okay.